test/test_intensify/test_intensify.py

```diff
@@ -1090,16 +1096,21 @@ def target(x):
         # to grab a new iteration, but all of them are exhausted
         # In this particular unit testing, no further new configurations
         # nor a chooser is provided, so we run into a value error
-        with self.assertRaisesRegex(ValueError, "No configurations/chooser provided"):
-            behest, run_info = intensifier.get_next_run(
-                challengers=[self.config1],
-                incumbent=inc,
-                run_history=self.rh,
-                chooser=None
-            )
+        intent, run_info = intensifier.get_next_run(
```

Comment:
The comment is no longer aligned with the code

Reply:
updated the comment to be:

 # In the upcoming get next run, the stage is RUN_CHALLENGER
 --        # so the intensifier tries to run config1. Nevertheless, 
 --        # there are no further instances for this configuration available. 
           # In this scenario, the intensifier produces a SKIP intent as an indication
 --        # that a new iteration must be initiated, and for code simplicity, 
           # relies on a new call to get_next_run to yield more configurations